client/selectors/index.js

```diff
@@ -170,15 +172,26 @@ export const getEventsWithMoreInfo = createSelector(
 * the associated events.
 */
 export const getEventsOrderedByDay = createSelector(
-    [getEventsWithMoreInfo],
-    (events) => {
+    [getCurrentSearch, getEventsWithMoreInfo],
+    (currentSearch, events) => {
         if (!events) return []
+        // check if search exists
+        let search = (currentSearch.advancedSearch || currentSearch.fulltext) ? true : false
```

Comment:
I would create directly a selector for that. Like `isASearchHappening`, but in a better English ^^). If the `currentSearch` definition changes one day, then we will avoid to go inside this `getEventsOrderedByDay` function to change it.